test/infrastructure/docker/Makefile

```
@@ -148,6 +148,7 @@ generate-manifests: $(CONTROLLER_GEN) ## Generate manifests e.g. CRD, RBAC etc.
 		paths=./api/... \
 		paths=./$(EXP_DIR)/api/... \
 		paths=./$(EXP_DIR)/controllers/... \
+		paths=./$(EXP_DIR)/internal/... \
```

```suggestion
		paths=./$(EXP_DIR)/internal/controllers/... \
```
I'll fixup the previously migrated controllers in https://github.com/kubernetes-sigs/cluster-api/pull/5696/files